app/assets/stylesheets/dm/pages/_partials/_header.scss

```diff
@@ -237,6 +247,10 @@
         right: 1rem;
       }
 
+      &#browse-by-locations-dropdown {
+               left: 26.125rem;
+      }
+
```

Comment:
Shouldn't this be `&#xr-network-dropdown`? `#browse-by-locations-dropdown` is already used below, so we have two elements with the same `id`. Also the dropdown looks a little off compared to the "Browse by locations" one. `left: 38.45rem` seems to match better.
![](https://user-images.githubusercontent.com/34111449/196782673-d3f07b20-0d1a-43ca-bec9-d7945d1ab42d.png)
![](https://user-images.githubusercontent.com/34111449/196782759-f1b1044f-2940-4aec-8bd3-c98fb1718dca.png)

Reply:
Yes good catch.. new styling settings look good!
![image](https://user-images.githubusercontent.com/60527788/196810085-cbca227b-ec39-4ec8-b7df-78c414e345ab.png)

Reply:
Yep - good catch.  got it.